CHANGES.rst

```diff
@@ -85,6 +85,12 @@ photom
   of NIRSpec IFU data with EXTENDED source type; correct units in pixel area
   division to sr from square arcseconds [#7336]
 
+ramp_fitting
+--------
+
+- Change the propagation of the SATURATED flag to be done only for complete
+  saturation. [#7363]
```

Comment:
Oh, and also reference the stcal PR number here, in addition to 7363.

Reply:
I linked the STCAL PR number here.